src/android/NetworkManager.java

```diff
@@ -289,46 +262,37 @@ private void sendUpdate(String type) {
      * @return the type of mobile network we are on
      */
     private String getType(NetworkInfo info) {
-        if (info != null) {
-            String type = info.getTypeName().toLowerCase(Locale.US);
-
-            LOG.d(LOG_TAG, "toLower : " + type.toLowerCase());
-            LOG.d(LOG_TAG, "wifi : " + WIFI);
-            if (type.equals(WIFI)) {
-                return TYPE_WIFI;
-            }
-            else if (type.toLowerCase().equals(TYPE_ETHERNET) || type.toLowerCase().startsWith(TYPE_ETHERNET_SHORT)) {
-                return TYPE_ETHERNET;
-            }
-            else if (type.equals(MOBILE) || type.equals(CELLULAR)) {
-                type = info.getSubtypeName().toLowerCase(Locale.US);
-                if (type.equals(GSM) ||
+        String type = info.getTypeName().toLowerCase(Locale.US);
+
+        LOG.d(LOG_TAG, "toLower : " + type);
+        LOG.d(LOG_TAG, "wifi : " + WIFI);
```

Comment:
Why do we even log the WIFI time after time. It just the constant wifi.
I think this can just be removed.

Reply:
Agreed.